src/ui/components/SecondaryToolbox/react-devtools/components/ReactDevToolsPanel.module.css

```diff
@@ -90,6 +90,7 @@
   gap: 0.5rem;
   padding: 0.5rem;
   border-radius: 0.25rem;
+  font-size: 1rem;
```

Comment:
We have variables for font size that we should be using. Probably `--font-size-regular` or `--font-size-regular-monospace`  for this?